R/test.R

```diff
@@ -64,16 +99,10 @@ runTests <- function(appDir=".", filter=NULL){
       warning("You've disabled `shiny.autoload.r` via an option but this is not passed through to shinytest. Consider using a _disable_autoload.R file as described at https://rstd.io/shiny-autoload")
```

Comment:
It would help if at line 90 (`if (all(isST))`) there was a comment that explained how we get into that code path -- that is, when the tests/ subdirectory uses the old structure, where it contains shinytest scripts.

Reply:
Turns out this was already illuminated, just higher up in the roxygen. I added a pointer to the code.